docs/src/languages/php/_index.md

```diff
@@ -304,16 +217,224 @@ markdownify=false
+| `create_function` | This function has been replaced by anonymous functions and shouldn't be used anymore.|
+| `exec`, `passthru`, `shell_exec`, `system`, `proc_open`, `popen` | These functions allow a PHP script to run a bash shell command. Rarely used by web apps except for build scripts that might need them. |
+| `pcntl_*` | The `pcntl_*` functions are responsible for process management. Most of them cause a fatal error if used within a web request. Cron tasks or workers may need them. Most are usually safe to disable. |
+| `curl_exec`, `curl_multi_exec` | These functions allow a PHP script to make arbitrary HTTP requests. If you're using HTTP libraries such as Guzzle, don't disable them. |
+| `show_source` | This function shows a syntax highlighted version of a named PHP source file. Rarely useful outside of development. |
+
+## Execution mode
+
+PHP offers two execution modes depending on your use-case:
+
+- Command line interface (PHP-CLI) is the mode used for command line scripts and standalone applications.
+    This is the mode used when logged into your container via SSH, for [crons](../../create-apps/app-reference.md#crons)
+    and usually also for [alternate start commands](#alternate-start-commands).
+    To use PHP-CLI mode, run your script with `php <SCRIPT>.php`.
+- Common gateway interface (PHP-CGI) is the mode used for web applications and web requests.
+    This is the default mode when the `start` command isn't explicitly set.
+    To use PHP-CGI mode, run your script with the symlink: `/usr/bin/start-php-app <SCRIPT>.php` instead of `php`.
+    PHP is run in CGI mode using PHP-FPM (FastCGI Process Manager).
+
+## Alternate start commands
+
+To specify an alternative process to run your code, set a `start` command.
+For more information about the start command, see [web commands reference](../../create-apps/app-reference.md#web-commands).
+
+By default, start commands use the PHP-CLI and run your script with `php <SCRIPT>.php`.
+Find out how and when to use [which execution mode](#execution-mode).
+
+Note that the `start` command must run in the foreground and is executed before the [deploy hook](../../create-apps/hooks/hooks-comparison.md).
+That means that PHP-FPM can't run simultaneously along with another persistent process such as [ReactPHP](https://github.com/platformsh-examples/platformsh-example-reactphp) or [Amp](https://github.com/platformsh-examples/platformsh-example-amphp).
+If you need multiple processes, they have to run in separate containers.
+
+Set up alternate start commands to:
+
+{{< codetabs >}}
+
+---
+title=Run a custom script
+file=none
+highlight=false
+---
+
+1. Add your script in a PHP file,
+2. Specify an alternative `start` command by adapting the variable:
+
+    ```yaml {location=".platform.app.yaml"}
+    web:
+        commands:
+            start: php <APP>.php
+    ```
+
+    Where `<APP>` is a file path relative to the [app root](../../create-apps/app-reference.md#root-directory).
```

Comment:
Similar issues as elsewhere (the name and the fact that its not the full file path).